pkg/cloud/cloud_test.go

```diff
@@ -3072,12 +3069,28 @@ func TestWaitForAttachmentState(t *testing.T) {
 	}
 }
 
+func noopVolumeWaitParameters() volumeWaitParameters {
```

Comment:
np: rename to `testVolumeWaitParameters`, as this isn't really "no-op" as `noop` would suggest